src/main/java/edu/ucsb/cs56/ucsb_open_lab_scheduler/controllers/ApplicationController.java

```diff
@@ -1,38 +1,31 @@
 package edu.ucsb.cs56.ucsb_open_lab_scheduler.controllers;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.stereotype.Controller;
-
-import java.util.HashMap;
-import java.util.Map;
-
+import edu.ucsb.cs56.ucsb_open_lab_scheduler.repositories.RoomAvailabilityRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
 import org.springframework.security.oauth2.client.registration.ClientRegistration;
 import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import edu.ucsb.cs56.ucsb_open_lab_scheduler.entities.Room;
-import edu.ucsb.cs56.ucsb_open_lab_scheduler.repositories.RoomRepository;
-import edu.ucsb.cs56.ucsb_open_lab_scheduler.entities.RoomAvailability;
-import edu.ucsb.cs56.ucsb_open_lab_scheduler.repositories.RoomAvailabilityRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.HashMap;
+import java.util.Map;
 
 @Controller
 public class ApplicationController{
-    private final RoomRepository roomRepository;
     private final RoomAvailabilityRepository RoomAvailabilityRepository;
 
     @Autowired
     private ClientRegistrationRepository clientRegistrationRepository;
 
     @Autowired
-    public ApplicationController(RoomRepository roomRepository, RoomAvailabilityRepository RoomAvailabilityRepository){
-        this.roomRepository = roomRepository;
+    public ApplicationController(RoomAvailabilityRepository RoomAvailabilityRepository){
         this.RoomAvailabilityRepository = RoomAvailabilityRepository;
     }
```

Comment:
Since when was the variable `RoomAvailabilityRepository` capitalized? I don't know if it's within the scope of this PR, but that should definitely be changed to lowerCamelCase...

Reply:
Fixed.